pandas/tests/test_categorical.py

```diff
@@ -3625,6 +3625,39 @@ def test_cat_accessor_api(self):
             invalid.cat
         self.assertFalse(hasattr(invalid, 'cat'))
 
+    def test_str_accessor_api_for_categorical(self):
+        # https://github.com/pydata/pandas/issues/10661
+        from pandas.core.strings import StringMethods
+        s = Series(list('aabb')).astype('category')
+        self.assertIsInstance(s.str, StringMethods)
+
+        res = s.str.contains("a")
```

Comment:
ok, that one found a bug (and a stupid guy in front of the keyboard who debugged 20min why `s.str.findall(" ") != c.str.split(" ")`...).

Anyway: under the condition that `take1d` and all the string methods are well tested, IMO this is now all tested as well: all new codepath are taken during the tests.